Quotient/connectionencryptiondata_p.cpp

```diff
@@ -422,7 +592,7 @@ void ConnectionEncryptionData::handleQueryKeys(const QueryKeysJob* job)
                   [this](const event_ptr_tt<EncryptedEvent>& pendingEvent) {
                       if (!isKnownCurveKey(
                               pendingEvent->fullJson()[SenderKey].toString(),
-                              pendingEvent->contentPart<QString>(SenderKeyKey)))
+                              pendingEvent->contentPart<QString>(SenderKey)))
```

Comment:
That can't be correct, can it? `SenderKeyKey == "sender_key"_ls` and `SenderKey == "sender"_ls`.
```suggestion
                              pendingEvent->contentPart<QString>(SenderKeyKey)))
```

Reply:
Fixed in a nicer way